src/meta/src/hummock/compaction/picker/min_overlap_compaction_picker.rs

```diff
@@ -238,7 +238,13 @@ impl NonOverlapSubLevelPicker {
 
             // more than 1 sub_level
             if ret.total_file_count > 1 && ret.total_file_size >= self.max_compaction_bytes
```

Comment:
for this picker, "ret.total_file_count > 1" is enough ? It means choosing more the 1 sub_level. However, both LGTM

Reply:
Why?
ret.total_file_count > 1 means that there may be only one sub-level with more than 1 files.

Reply:
We choose the basic sst in the input level (at least one), So, when sst in ret is greater than 1, it means we select other sub-level